### ISSUE TITLE
Question about the performance of GRU module

### ISSUE THREAD

**Issue by vincentgc** [mkocabas/VIBE#13]
Hi, 
I set 'disable_temporal' to 'True' when I define the VIBE_Demo model to test the performance of GRU module, but there was no difference in effect compared to when I did not set this option. Is this GRU module really useful? And I didn't see the ablation experiments in the paper. Do you have relevant ablation experiments to verify that the GRU module is effective? :)

**Comment by mkocabas** [comment]
Hi @vincentgc,

Thanks for your interest!

`disable_temporal` option is identical to SPIN inference, so the results you get is the same exact results of SPIN. So, GRU module is useful indeed as shown with the experiments we compare against SPIN, please refer to Table 2 in the [paper ](https://arxiv.org/pdf/1912.05656.pdf) or the [evaluation table](https://github.com/mkocabas/VIBE#evaluation) in the readme. You would be able to validate these results once we release the evaluation and training codes. Besides, we observe VIBE produces smoother and better results in certain cases.

**Comment by ikvision** [comment]
@mkocabas  
Does it mean that during VIBE training HMR and SMPL building blocks (SPIN part) have zero learning rate?

**Comment by mkocabas** [comment]
We precompute the image features using ResNet50 initialized with SPIN checkpoint. Then, we use these features during training, hence ResNet50 isn't updated. But, all other blocks i.e. GRU layers and SMPL regressor are trained. Please refer to Section 3.3 in the paper.
